src/modules/history/HistoryModule.ts

```diff
@@ -102,21 +102,25 @@ export class HistoryModule implements ReaderModule {
           " disabled",
           ""
         );
+        this.historyForwardAnchorElement.hidden = false;
```

Comment:
Ok, everything else works as it's supposed to, but i would rather not hide the forward or backward buttons, as these are used in the example, for the implementation any UI can be used other than the sample UI

Reply:
Sure, just pushed a commit so that they are no longer being hidden.  It just made it easier to test.